### PR TITLE
test(Intl.DurationFormat.prototype): check prototype

### PR DESCRIPTION
**Pull request by EarthyOrange**
#3305

### CODE REVIEW

**Comment by rwaldron** [comment]
@EarthyOrange thanks for the contribution! According to CI, there is an issue here that's easily resolvable and I'll leave a comment in-line to show what needs to be fixed.

File: test/intl402/DurationFormat/prototype/prototype_attributes.js [hunk 1]
```diff
@@ -0,0 +1,15 @@
+// Copyright (C) 2021 Nikhil Singhal. All rights reserved.
+// This code is governed by the BSD license found in the LICENSE file.
+
+/*---
+esid: sec-Intl.DurationFormat.prototype
+description: Prototype attributes verification
+info: |
+  This property has the attributes { [[Writable]]: false, [[Enumerable]]: false, [[Configurable]]: false }.
+features: [Intl.DurationFormat.prototype]
```
Comment (rwaldron) [inline test/intl402/DurationFormat/prototype/prototype_attributes.js:9]:
```suggestion
features: [Intl.DurationFormat]
```

File: test/intl402/DurationFormat/prototype/toStringTag.js [hunk 2]
```diff
@@ -0,0 +1,30 @@
+// Copyright (C) 2021 Nikhil Singhal. All rights reserved.
+// This code is governed by the BSD license found in the LICENSE file.
+
+/*---
+esid: sec-Intl.DurationFormat.prototype-@@tostringtag
+description: Checks prototype's toStringTag value
+info: |
+  The initial value of the @@toStringTag property is the string value "Intl.DurationFormat".
+  This property has the attributes { [[Writable]]: false, [[Enumerable]]: false, [[Configurable]]: true }.
+features: [Symbol.toStringTag]
```
Comment (rwaldron) [inline test/intl402/DurationFormat/prototype/toStringTag.js:10]:
```suggestion
features: [Intl.DurationFormat, Symbol.toStringTag]
```

**Review by rwaldron — CHANGES_REQUESTED** [review]
In addition to those changes, please add a new feature to https://github.com/tc39/test262/blob/main/features.txt, which should be: 

```
# Intl.DurationFormat
# https://github.com/tc39/proposal-intl-duration-format
Intl.DurationFormat
```

**Comment by EarthyOrange** [comment]
> In addition to those changes, please add a new feature to https://github.com/tc39/test262/blob/main/features.txt, which should be:
> 
> ```
> # Intl.DurationFormat
> # https://github.com/tc39/proposal-intl-duration-format
> Intl.DurationFormat
> ```

Did adding these lines to the features.txt appease the CI?

**Comment by rwaldron** [comment]
@EarthyOrange 
> Did adding these lines to the features.txt appease the CI?

According to the CI report, it sure did!

**Review by rwaldron — APPROVED** [review]

**Pull request merged**